### PR TITLE
Scipy structure refinement

### PR DESCRIPTION
**Pull request by karl9000**
The content is not changed too much. In total, a section-based structure is introduced and a bit more descriptive text is added.

### CODE REVIEW

File: python/scientific-python.ipynb [hunk 1]
```diff
@@ -13,18 +13,24 @@
     "\n",
     "Überblick über (fast) alles weitere was SciPy kann gibt es hier:\n",
     "\n",
-    "- https://github.com/MaxNoe/scientific_python_notebooks/blob/master/scipy.ipynb"
+    "- https://github.com/MaxNoe/scientific_python_notebooks/blob/master/scipy.ipynb\n",
+    "\n",
+    "Diese Einführung konzentriert sich auf die folgenden Themen:\n",
+    "\n",
+    "- Abrufen von physikalischen Naturkonstanten samt Unsicherheit und Einheit\n",
+    "- Lineare und Nichtlineare Regression"
```
Comment (chrbeckm) [inline python/scientific-python.ipynb:21]: In the python-notebook, we use a table of content.
Use 
```
%%javascript
$.getScript('https://kmahelona.github.io/ipython_notebook_goodies/ipython_notebook_toc.js')
```
in a Code cell and
```
# Inhalt

<div id="toc"></div>
```
in a Markdown cell.

File: python/scientific-python.ipynb [hunk 2]
```diff
@@ -13,18 +13,24 @@
...
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": null,
+   "cell_type": "markdown",
    "metadata": {},
-   "outputs": [],
    "source": [
-    "import numpy as np\n",
```
Comment (chrbeckm) [inline python/scientific-python.ipynb:25]: Numpy is needed later on!

File: python/scientific-python.ipynb [hunk 3]
```diff
@@ -447,6 +430,68 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "x = np.array([2, 5, 7])  # create test array\n",
+    "\n",
+    "from scipy.stats import gmean, sem\n",
```
Comment (chrbeckm) [inline python/scientific-python.ipynb:456]: `gmean` is unused

**Review by dormail — CHANGES_REQUESTED** [review]
Need to add stuff to lecture frame "What is Unix?"

**Comment by maxnoe** [comment]
Why include unix stuff at all in a PR about scipy. Please just open another one and remove the changes here.

**Review by maxnoe — APPROVED** [review]

**Review by maxnoe — APPROVED** [review]
Thanks! I just added the javascript call to actually fill the toc

**Pull request merged**